README.md

```diff
@@ -39,11 +39,11 @@ export DYNAFLOW_LAUNCHER_LOCALE=en_GB
 # export DYNAFLOW_LAUNCHER_LOG_LEVEL=INFO # default INFO: can be DEBUG, INFO, WARN, ERROR
 
 # Optional external links : optional variables used to add other models to DynaFlow simulation
-# DYNAFLOW_LAUNCHER_EXTERNAL_HOME=<PATH_TO_EXTERNAL_INSTALL>
-# export DYNAFLOW_LAUNCHER_EXTERNAL_DDB=$DYNAFLOW_LAUNCHER_EXTERNAL_HOME/ddb
+# DYNAWO_LAUNCHER_EXTERNAL_HOME=<PATH_TO_EXTERNAL_INSTALL>
```

Comment:
"LAUNCHER" looks weird when it is DYNAWO. Maybe just DYNAWO_EXTERNAL_HOME ?

Reply:
yes, sorry, wrong replacement :)